csharp/private/rules/nunit_test.bzl

```diff
@@ -64,6 +65,10 @@ csharp_nunit_test = rule(
             doc = "A list of analyzer references.",
             providers = AnyTargetFramework,
         ),
+        "keyfile": attr.label(
```

Comment:
I can't imagine strong-naming test projects, despite it being possible. Is this something you actually do, or just being consistent?

Reply:
Mostly just being consistent here. I can't imagine doing it either, but there might be some cases where the user would want to do it.

Reply:
MSBuild allows it too ;)

Reply:
Is NUnit and NUnit Lite strong-named?

Reply:
Good question. Maybe? Random commit: https://github.com/nunit/nunit/commit/ebf531ad5bce76f3358755b4c4a304c55a2b7363

Reply:
I'm not sure. Looking at the binaries with ildasm, they don't appear to be, though I'm not certain it matters.

Reply:
Looks like it

![image](https://user-images.githubusercontent.com/597907/65802686-6895fb80-e14a-11e9-8193-34298bf2181d.png)

Reply:
(Er, I'm assuming that PublicKey stuff is about signing... I don't actually know how this stuff works :) )

Reply:
I'm thinking the strong named test assembly won't be able to reference / load the not-strongly-named NUnit?

I may not understand strong naming though.

Reply:
> I'm assuming that PublicKey stuff is about signing

Yeah, I'd think so